main.go

```diff
@@ -1,227 +1,200 @@
 package fixtures
 
 import (
-	"github.com/src-d/lookout-sdk/pb"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+
+	"github.com/google/go-github/github"
 )
 
 var fixtures = Fixtures{
 	{
 		Name: "new-go-file-too-long-line",
-		CommitRevision: pb.CommitRevision{
```

Comment:
can we have methods for it then? Take a look for example here: https://github.com/src-d/lookout/pull/243

I don't care about github there at all. I don't want to test/rely on `castPullRequest` method for that tests. I just want to have correct base&head.

Reply:
Done in 94ea632